Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenuNavViewTemplate.xaml

```diff
@@ -161,6 +161,11 @@
                             IsSettingsVisible="False"
                             IsPaneToggleButtonVisible="{Binding HamburgerVisibility, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource VisibilityToBoolConverter}}"
                             Header="{x:Null}">
+                <NavigationView.HeaderTemplate>
+                    <DataTemplate>
+                        <Grid></Grid>
```

Comment:
Won't this require users to set a HeaderTemplate? Meaning that even if they set a Header that it will not show?

Reply:
The HamburgerMenu doesn't have a header, there is no way to modify it without re-templating it anyway

Reply:
So this wouldn't be needed then. If the Header of the NavigationView isn't being set there is no need for a HeaderTemplate

Reply:
Just tried it. Looks like the NavigationView has a default template for the Header, so if you don't specify anything, it binds a TextBlock to the DataContext by default

Reply:
That's standard. But the Content of the HeaderTemplate should be set to the Header itself
```
<ContentPresenter Content="{TemplateBinding Header}" ContentTemplate="{TemplateBinding HeaderTemplate}"/>
```

Reply:
Not sure what you mean, the HamburgerMenu doesn't have a Header and HeaderTemplate

Reply:
Yes, that would be for the NavigationView. There shouldn't be any reason to set an empty header template because the header content would be null

Reply:
I'm probably doing something wrong, I haven't been able to get it work any other way other then setting a blank template. Setting Header to null doesn't work. Setting header to an empty grid doesn't work.